application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -36,26 +36,40 @@
   <minorEdit>false</minorEdit>
   <syntaxId>plain/1.0</syntaxId>
   <hidden>true</hidden>
-  <content># Class fields
-Analytics.Code.AnalyticsDashboardClass_startDate=Start Date
-Analytics.Code.AnalyticsDashboardClass_endDate=End Date
+  <content>## Class fields
+Analytics.Code.AnalyticsDashboardClass_description.hint=Additional information to be displayed.
 Analytics.Code.AnalyticsDashboardClass_description=Description
-Analytics.Code.AnalyticsDashboardClass_displayedStatistics=displayed Statistics
-Analytics.Code.AnalyticsDashboardClass_startDate.hint=Start time of the date interval used on all statistics of this dashboard.
+Analytics.Code.AnalyticsDashboardClass_displayedStatistics=Displayed Statistics
+Analytics.Code.AnalyticsDashboardClass_endDate=End Date
 Analytics.Code.AnalyticsDashboardClass_endDate.hint=End time of the date interval used on all statistics of this dashboard.
-Analytics.Code.AnalyticsDashboardClass_description.hint=Additional information to be displayed.
 Analytics.Code.AnalyticsDashboardClass_DisplayedStatistics.hint=List of statistics widgets that will be displayed in the current dashboard, in order.
+Analytics.Code.AnalyticsDashboardClass_startDate=Start Date
```

Comment:
Just a small comment, but you don't need to capitalize words that are not at the beginning of the sentence.